### ISSUE TITLE
Application displays duplicate entries for tutorial sections

### ISSUE THREAD

**Issue by scriptbae** [inspectorG4dget/AnonymousFeedback#34]
_This issue is suspected to be related to #33._

Using the current master:HEAD (5e692f20a11302b906aceb5d1b61d6f5b48758d7) as well as the provided database bootstrapping file and testing data, the application displays duplicates of tutorial sections when attempting to evaluate a TA for course `ITI1121`. Related screenshot attached.

![screenshot from 2017-01-24 22-58-08](https://cloud.githubusercontent.com/assets/1712054/22277334/b3aa85f0-e288-11e6-9b53-2fc570496dca.png)

The cause of this issue requires further investigation.

**Comment by scriptbae** [comment]
This issue is no longer suspected to be related to #33.

**Comment by cannotparse** [comment]
The bug isn't fixed. Bug appears on my version too.

**Comment by cannotparse** [comment]
I believe this was apart of a feature I was half way through implementing on the day before we needed to "present" it. 
Let me bring up the code.

**Comment by scriptbae** [comment]
@NuclearBanane brain fart. The bug is indeed still present.

**Comment by cannotparse** [comment]
Below code needs to have the exception handling fixed. But it was trying to differentiate the context of needing to only see all the sections in existence to assign TA's to it in the assign TA form. This way this function can server both getting active sections with TA's to the student view and in-active sections still requiring sections. Didn't mange to complete it in time, ofc. 
```python
try:
     active = self.get_argument('active')
except :
     active = False
```
You'll also want to look at `def getSections(courseCode, year, semester, active):`

**Comment by scriptbae** [comment]
@NuclearBanane that snippet seems to be unrelated.

**Comment by cannotparse** [comment]
It isn't, it chooses between the following queries which need to be fleshed out.    
```python
if active is True:
        t.execute("""SELECT sectionID, weekday, startTime, endTime
                FROM section, teaches
                WHERE section.course=%s
                    AND section.currYear=%s
                    AND section.semester=%s
                    AND teaches.currYear=section.currYear
                    AND teaches.semester=section.semester
                    AND teaches.course=section.course""",
                    (courseCode, year, semester))
    else :
        t.execute("""SELECT sectionID, weekday, startTime, endTime
                FROM section
                WHERE course=%s
                    AND currYear=%s
                    AND semester=%s""",
                    (courseCode, year, semester))
```

**Comment by scriptbae** [comment]
Tentatively fixed in PR #40. Will put together some test cases later tonight and if that merge passes the test cases, this issue will be closed.
